pages/architecture/distributed_registry.rst

```diff
@@ -0,0 +1,230 @@
+.. _nuts-documentation-architecture-certificates:
+
+Distributed Registry
+#########################
+
+The goal of this system is to provide a registry in which parties can lookup endpoints which can be used to exchange data.
+
+Non-functional Requirements
+***************************
+
+Availability
+------------
+
+* Every consumer of the registry should have its own instance, because:
+  * End users will be using the registry to search for care organisations, probably using full-text search. Having a local copy gives the best performance and user experience.
+  * Remote systems and networks can be unreliable, having a local copy ensures availability.
+* Every instance should connect to other instances and be open for connections from other instances, which provides
+  better availability than relying on a centralized instance.
+
+Durability
+----------
+
+* Every instance should maintain a complete copy of the registry (including the complete history), and be willing
+  to share it with other instances. This helps to mitigate the risk of data loss.
+
+Security
```

Comment:
The header `Security` currently describes requirements about integrity.

Reply:
Security as 'umbrella-term' (integrity, confidentiality, etc)